File: Nível 1/1865 - Mjölnir/main.py
valor = int(input("A: "))

for i in range(1, valor + 1):
    texto = input("B: ")
    x1,x2 = texto.split(" ")
    
    x2 = int(x2)
    
    x1 = x1.lower()
    
    if x1 == "thor":
        print("Y")
    else:
        print("N")

File: Nível 1/1865 - Mjölnir/main.lua
valor = tonumber(io.read())

for i = 1, valor do 
    texto = io.read()
    x1, x2 = texto:match("(%S+)%s+(%S+)")
    x2 = tonumber(x2)
    x1 = string.lower(x1)

    if x1 == "thor" then
        print("Y")
    else
        print("N")
    end
end